content/get-started/quickstart/be-social.md

```diff
@@ -43,39 +45,33 @@ Click **Watch** at the top of a repository to watch it.
 
- You may want to contribute to another person's project, whether to add features or to fix bugs. After making changes, let the original author know by sending a pull request. For more information, see "[About pull requests](/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests)."
+For an overview of these tools and information about the specificity of each, see "[Quickstart for communicating on {% data variables.product.prodname_dotcom %}](/github/collaborating-with-issues-and-pull-requests/quickstart-for-communicating-on-github)." To learn more about discussions, see "[{% data variables.product.prodname_dotcom %} Discussions](https://docs.github.com/en/discussions)."
 
- ![Pull request button](/assets/images/help/repository/repo-actions-pullrequest.png)
+## Participating in organizations
 
-### Using issues
+Organizations are shared accounts where businesses and open-source projects can collaborate across many projects at once. Owners and administrators can establish teams with special permissions, have a public organization profile, and keep track of activity within the organization. For more information, see "[About organizations](/articles/about-organizations/)."
 
-When collaborating on a repository, use issues to track ideas, enhancements, tasks, or bugs. For more information, see '[About issues](/articles/about-issues/)."
+From your dashboard, click the drop down menu of your username on the left side of your dashboard. Here, you can view organizations that you are a part of and easily switch between them.
 
-![Issues button](/assets/images/help/repository/repo-tabs-issues.png)
+![Switch account context dropdown](/assets/images/help/overview/dashboard-contextswitcher.png)
 
-### Participating in organizations
+### Exploring other projects on {% data variables.product.prodname_dotcom %}
 
-Organizations are shared accounts where businesses and open-source projects can collaborate across many projects at once. Owners and administrators can establish teams with special permissions, have a public organization profile, and keep track of activity within the organization. For more information, see "[About organizations](/articles/about-organizations/)."
+You can discover new and interesting projects on {% data variables.product.prodname_dotcom %}'s Explore page. Your dashboard feed will generate projects based on your interests from repositories you have starred, people you follow, and your public repository contributions. From your dashboard, you can see trending projects, view topics, and check out collections. Star interesting projects and come back to them later. Visit your stars page to see all your starred projects. For more information about stars, see "[Saving repositories with stars](/get-started/exploring-projects-on-github/saving-repositories-with-stars)."
```

Comment:
I love the changes you've made here but think it would benefit from a little rearrangement. Something like?

```suggestion
You can discover new and interesting projects on {% data variables.product.prodname_dotcom %}'s Explore page. You can star interesting projects to make them easy to find again later. Visit your stars page to see all your starred projects. For more information about stars, see "[Saving repositories with stars](/get-started/exploring-projects-on-github/saving-repositories-with-stars)."

Your dashboard feed displays projects based on your interests including repositories you star, people you follow, and your contributions to public repositories. From your dashboard, you can see trending projects, view topics, and check out collections. 
```